src/xenia/kernel/xam/apps/xgi_app.cc

```diff
@@ -147,6 +147,26 @@ X_HRESULT XgiApp::DispatchMessageSync(uint32_t message, uint32_t buffer_ptr,
       XELOGD("XGI_unknown");
       return X_STATUS_SUCCESS;
     }
+    case 0x000B0021: {
+      // Games used in:
+      // - Fatal Fury Special, Beijing 2008, 50 Cent: Blood on the Sand,
```

Comment:
Use title_id instead of name, just in case

Reply:
Looks like you just added it. Will remove now